Assets/Scripts/Utils/Vector.cs

```diff
@@ -3,11 +3,11 @@
 namespace Utils {
     public abstract class Vector {
         public static Vector3 GetRandomPosition(float dx, float dy, float dz) {
-            float x = Random.Range(-dx, dx);
-            float y = Random.Range(-dy, dy);
-            float z = Random.Range(-dz, dz);
-
-            return new Vector3(x, y, z);
+            Vector3 rdPosition = Random.onUnitSphere;
+            rdPosition.x *= dx;
+            rdPosition.y *= dy;
+            rdPosition.z *= dz;
+            return rdPosition;
```

Comment:
![imo](https://img.shields.io/badge/review-imo-yellow.svg) 

I'm thinking about `Random.onUnitSphere` for `GetRandomDirection()`
but this implementation is also ok

Reply:
and I just thought more short code

```suggestion
Vector3 scalingVector = new Vector3(dx, dy, dz)'

return Vector3.Scale(Random.onUnitSphere, scalingVector)
```

Reply:
ah my bad, didn't pay attention to which function aha